src/components/KDateTimePicker/CalendarWrapper.vue

```diff
@@ -104,6 +100,32 @@ const originalTimeValues = ref<{ start: string, end: string }>({
 })
 const componentId = useId()
 
+const initialPage = computed(() => {
+  if (props.isRange && calendarVModel.value && 'start' in calendarVModel.value && calendarVModel.value.start instanceof Date) {
+    return { year: calendarVModel.value.start.getFullYear(), month: calendarVModel.value.start.getMonth() + 1 }
+  } else if (calendarVModel.value instanceof Date) {
+    return { year: calendarVModel.value.getFullYear(), month: calendarVModel.value.getMonth() + 1 }
+  } else if (props.maxDate) {
+    return { year: props.maxDate.getFullYear(), month: props.maxDate.getMonth() + 1 }
```

Comment:
I think we should always prefer current date over `maxDate` unless it's already beyond `maxDate`.

Reply:
I agree, pushed a change